waku/waku_api/rest/admin/openapi.yaml

```diff
@@ -49,6 +49,24 @@ paths:
           description: Cannot connect to one or more peers.
         '5XX':
           description: Unexpected error.
+  /admin/v1/filter:
```

Comment:
Would recommend appending `/subscriptions` to this endpoint. 
This would help in future to add some other admin sub endpoints for filter.

Reply:
Great point! Will add it now :))
Thanks!